index.html

```diff
@@ -291,6 +303,63 @@
           </dt>
         </dl>
       </section>
+      <section>
+	<h2>
+	  Media Devices
+	</h2>
+	<p>
+	  The <code><dfn id='idl-def-PermissionName.camera'>camera</dfn></code>,
+	  <code><dfn id='idl-def-PermissionName.microphone'>microphone</dfn>
+	  </code>,
+	  and <code><dfn id='idl-def-PermissionName.speaker'>speaker</dfn>
+	  </code>
+	  permissions are associated with permission to use media
+	  devices as specified in [[GETUSERMEDIA]] and
+	  [[audio-output]].
+	</p>
+	<p>These three permissions have an associated PermissionDescriptor:
+	</p>
+	<dl class='idl' title='dictionary DevicePermissionDescriptor :
+	    PermissionDescriptor'>
+	  <dt>
+	    DOMString deviceId
+	  </dt>
+	</dl>
+	<p>A permission covers access to the device given in the
+	associated descriptor.
+	</p>
+	<p>
+	  If the descriptor does not have a deviceId, its semantic is that
+	  it queries for access to any device of that class.
+	</p>
+	<p>
+	  If the deviceId is the special string "*", it queries for
+	  access to all devices of that class, whether they exist
+	  at the moment or are attached at a later time. When such a
+	  permission has been granted, queries for permission to any
+	  specific device MUST return "granted" as long as the
+	  deviceId is the ID of a connected device of the given class.
```

Comment:
You don't explain what to do in the case of `deviceId` being something else. Is that on purpose. I particularly wonder what we should do if it is the empty string.